react/features/settings/actions.native.ts

```diff
@@ -36,3 +36,16 @@ export function openLogoutDialog() {
         }));
     };
 }
+
+/**
+* Changes the display name for the local user.
+*
+* @param {string} _nickname - The new display name.
+* @returns {Function}
+*/
+export function changeLocalDisplayName(_nickname = '') {
+    // not used on mobile.
```

Comment:
Mobile does have the ability to change the display name in the settings pane, so looks like we are duplicating some logic here.

Reply:
It seems it is doing the same thing. It dispatches the updateSettings action with the new value. It just doesn't call the `getNormalizedDisplayName` function.